docroot/modules/humsci/hs_bugherd/src/ConfigOverrides.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Drupal\hs_bugherd;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\encrypt\EncryptService;
+use Drupal\encrypt\Entity\EncryptionProfile;
+
+/**
+ * Example configuration override.
+ */
+class ConfigOverrides implements ConfigFactoryOverrideInterface {
+
+  /**
+   * @var \Drupal\Core\Extension\ModuleHandlerInterface
+   */
+  protected $moduleHandler;
+
+  /**
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * @var \Drupal\encrypt\EncryptService
+   */
+  protected $encryption;
+
+  /**
+   * ConfigOverrides constructor.
+   *
+   * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   * @param \Drupal\encrypt\EncryptService $encrypt_service
+   */
+  public function __construct(ModuleHandlerInterface $module_handler, ConfigFactoryInterface $config_factory, EncryptService $encrypt_service) {
+    $this->moduleHandler = $module_handler;
+    $this->configFactory = $config_factory;
+    $this->encryption = $encrypt_service;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function loadOverrides($names) {
```

Comment:
Inherit doc is nice, but it would also be nice if you stated somewhere what this is overriding and why.